frontend/app/package.json

```diff
@@ -113,7 +113,7 @@
     "unplugin-auto-import": "0.15.3",
     "unplugin-vue-components": "0.24.1",
     "unplugin-vue-define-options": "1.3.3",
-    "vite": "4.2.2",
+    "vite": "4.3.1",
```

Comment:
I update it back to 4.3.1, since we last tested with this version. (Latest is 4.3.7)